artifacts/ArtifactHandling/Import-AppArtifact.ps1

```diff
@@ -99,6 +101,28 @@ function Import-AppArtifact {
                 $success = $true
             }
     
+            if ($scope -eq 'Dev') {
```

Comment:
I'm not sure if it's such a good idea to handle the dev endpoint and then simply exit the function with a return.

The log functionality is also not being used...
@cosmoconsult/ppi what do you think?

Reply:
Thanks for the explanation.